src/components/button/Button.jsx

```diff
@@ -0,0 +1,25 @@
+import './button.css';
+
+function Button({ type, fruit, textContent, formState, handleChange }) {
+    return (
+        <button
+            type={type}
+            textContent={textContent}
+            formState={formState}
+            name={type === 'button' ? fruit : `${type}-button`}
+            // Plus and minus button
+            disabled={textContent === '-' && formState[fruit] === 0}
+            onClick={
+                (type === 'button') ? ((textContent === '-') ? (e) => handleChange(e, formState[fruit] - 1) : (e) => handleChange(e, formState[fruit] + 1)) :
```

Comment:
```suggestion
                (type === 'button') ? (e) => handleChange(e, formState[fruit] + (textContent === '-' ? - 1 : 1)) :
```
Is iets makkelijker te lezen en je code is een stukje korter 😉